lib/eblox/data/post.ex

```diff
@@ -86,22 +86,20 @@ defmodule Eblox.Data.Post do
     end
   end
 
-  @behaviour Siblings.Worker
-
-  @impl Siblings.Worker
-  def perform(:idle, _id, _payload) do
-    {:transition, :read, nil}
+  @impl Finitomata
+  @doc false
+  def on_transition(:parsed, :update, event_payload, payload) do
+    {:ok, :parsed, Map.put(payload, :data, event_payload)}
```

Comment:
What this `data` is for? Do you have plans for it other than testing?

Reply:
Answered above, but I think, you’re right. Let’s remove it now, assert on messages sent from `on_enter/2`, and keep in mind we might have the payload updated that way later.